pyproject.toml

```diff
@@ -9,20 +9,20 @@ authors = [
 repository = "https://github.com/LyntServices/greykite.git"
 
 [tool.poetry.dependencies]
-python = "^3.10,<3.13"
-cvxpy = "1.2.1"
+python = "^3.11,<3.12"
+cvxpy = "1.3"
 dill = "^0.3.6"
 holidays = "0.13"
 holidays-ext = "^0.0.8"
 ipython = "^7.31.1"
 matplotlib = "3.8.1"
-numpy = "^1.23.2,<1.25.0"
+numpy = "1.26.0"
 osqp = "^0.6.3"
 overrides = "^7.3.1"
-pandas = "2.1.0"
+pandas = "2.1.1"
 patsy = "^0.5.2"
 plotly = "^5.4.0"
-pmdarima = "1.8.5"
+pmdarima = "2.0.2"
```

Comment:
Je nutné zde používat konkrétní verze u knihoven `cvxpy`, `numpy`, `pandas` a `pmdarima`? Nešlo by podmínky uvolnit alespoň na možnost použít libovolnou novější patch verzi, tedy např. na 2.1.* v případě `pandas`?

V poetry k tomu lze využít "tilde requirement": https://python-poetry.org/docs/dependency-specification/#tilde-requirements

Reply:
Prošla jsem si pro použité minor verze jejich patch verze, a můžeme je uvolnit, s výjimkou cvxpy, které nemá patch verze. Zatím se trochu bojím moc uvolňovat, ještě jsem nějak detailně nečetla, jak pandas rozhoduje, kterou verzi přesně z povolených nainstaluje.